pkg/zrouter/zmiddlewares/cache.go

```diff
@@ -38,6 +40,19 @@ func CacheMiddleware(cache zcache.ZCache, config domain.CacheConfig) func(next h
 	}
 }
 
+func matchPathWithConfig(path string, configPaths map[string]time.Duration) (time.Duration, bool) {
+	for configPath, ttl := range configPaths {
+		escapedConfigPath := regexp.QuoteMeta(configPath)
+		regexPattern := strings.Replace(escapedConfigPath, "\\{address\\}", "[^/]+", -1)
+		regex := regexp.MustCompile("^" + regexPattern + "$")
+
+		if regex.MatchString(path) {
+			return ttl, true
+		}
+	}
+	return 0, false
+}
```

Comment:
This is too specific for 1 kind of parameter. What if we replace anything inside curly braces ( { } ) instead of just the {address}? The middleware is inside golem, so it should be pretty much generic. Or maybe we could say something like "if the configure path has an *, like `transactions/*/latest, accept anything there"... or even better, the configured paths are regex themselves....